src/stories/Dropdown.stories.tsx

```diff
@@ -0,0 +1,150 @@
+import type { Meta, StoryObj } from '@storybook/react';
+import React from 'react';
+import { ArrowRightOnRectangleIcon } from '@heroicons/react/24/outline';
+
+import {
+  Dropdown,
+  DropdownTrigger,
+  DropdownMenu,
+  DropdownMenuTitle,
+  DropdownMenuLink,
+} from '../components/Dropdown';
+
+const meta = {
+  component: Dropdown,
+  tags: ['autodocs'],
+  // Storybook elements are placed inside `overflow:hidden` containers,
+  // which in our case hide the dropdown menu. This lets us see them,
+  // at the expense of space.
```

Comment:
Did you mean to put this comment next to the `decorator`? Or what is the "this" that lets us seem them?

Reply:
Oh, yes, thank you! This was leftover from when I originally defined `decorators` here. But then I realized each example needed its own `height`, and so had to define them individually. (I didn't try very hard to have one default decorator, over-rideable by individual examples; it seemed to just nest them, which didn't work.)

Let me take one more look at if I can override a default decorator, and if not, I'll move this comment to the `Default` example.

Reply:
I suspect there is a solution to what I want (default component decorator that individual stories can override) either through the use of custom story arguments or some sort of "story decorator overrides component decorator" feature. But I'm not finding it quickly enough and that's really just an optimization so I'm going to go with the latter solution above.